### ISSUE TITLE
Remove MongoDB

### ISSUE THREAD

**Issue by luistrigueiros** [microsoft/TypeScript-Node-Starter#218]
I think that this starter should be made simpler without including MongoDb upfront and have a generator.
As a starter I am looking for guidance on how to setup Node+Express+Typescrit but not Mongo.
More important are things like linter and how to install typedefintions and maybe connect to an external API.

**Comment by orta** [comment]
Thanks Oscar, different people are coming looking for different things - Given that there's a focus on building a node API here, I think conforming to the MEAN stack makes a lot of sense for this template.

**Comment by luistrigueiros** [comment]
As a starter template for people coming new to the language you should make it simpler and more accessible, maybe by using an embedded mongo so that people new should not go about also setup mongo.

**Comment by victorsebrito** [comment]
I think Microsoft should provide more starter projects for TypeScript. I'd love to find an official template for simple TS library or a CLI application.

**Comment by orta** [comment]
I'd recommend looking at tsdx  for that - https://github.com/palmerhq/tsdx

**Comment by jkjha** [comment]
Yes, please remove MongoDB. Starter project should be like template for new project.

**Comment by malforsaja** [comment]
If you check branch "noDatabase" of this template MongoDB is removed.

**Comment by geoffwellman** [comment]
I agree, it's not a great "starter" when it forces you to use a database, this boilerplate has got very bloated from what it used to be.

**Comment by diberry** [comment]
Can the starter move toward feature flags and modules - that would allow the features to grow separate from but inclusive to the starter.

**Comment by samal-rasmussen** [comment]
The complexity of maintaining the mongo db integration might be one of the reasons holding back the maintenance of this project. Currently TypeScript-Node-Starter is falling way behind in upgrading the packages in package.json. See: https://github.com/microsoft/TypeScript-Node-Starter/issues/305

**Comment by top-master** [comment]
Maybe at least, just replace MongoDB with MySQL;

No one did outlaw developing accounting App with `Node.js` ;-)

Like your fork: https://github.com/akitoeki/TypeScript-Express-MySQL-Starter
